### PR TITLE
[Sugar Cube] Create the prompt UI which show full length #84

### LINKED ISSUE #84: [Sugar Cube] Create the prompt UI which show full length

### PR DESCRIPTION
**Pull request by LovedeepSingh19**
Issue -- #84
added an arrow icon which show prompt Template on Sugar Cubes page.
It fetches prompt Template text from api '''api.cube.getPrompt.useQuery()'''
Screenshot 2023-11-17 at 1 42 00 AM
Screenshot 2023-11-17 at 1 42 19 AM

### CODE REVIEW

**Comment by devilankur18** [comment]
@LovedeepSingh19 move it to seperate compinent.

**Comment by LovedeepSingh19** [comment]
@devilankur18 can plz you check and let me know if this works. If not then please clarify what you meant by "move it above variables".

**Comment by devilankur18** [comment]
@LovedeepSingh19 please place that component ablove the variables text in the UI.

**Comment by devilankur18** [comment]
@all-contributors please add @LovedeepSingh19 for code contributions

File: apps/factory/src/components/prompt_view_arrow.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { Stack, Box, Typography } from "@mui/material";
+import { FaCaretDown, FaCaretUp } from "react-icons/fa";
+
+type Prompt_view_arrowProps = {
+  PromptTemplate: string;
+};
+
+const Prompt_view_arrow: React.FC<Prompt_view_arrowProps> = ({
```
Comment (devilankur18) [inline apps/factory/src/components/prompt_view_arrow.tsx:9]: @LovedeepSingh19 please following the naming convention as per other components. it should be camel cased

**Review by devilankur18 — CHANGES_REQUESTED** [review]

File: apps/factory/src/components/prompt_view_arrow.tsx (hunk 1 above)
Comment (LovedeepSingh19) [reply apps/factory/src/components/prompt_view_arrow.tsx]: Done👍🏻, Sorry about that.
Screenshot 2023-11-17 at 6 19 19 PM
is this what you ment by "Just place that component above the variables text in the UI."

**Pull request merged**